views/index.ejs

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
You don't need the html layout here because it's taken care of by the layout file